packages/core/src/utils/createHash.ts

```diff
@@ -0,0 +1,6 @@
+import { anyToString } from './stringUtils';
+import { createHash as createHashRaw } from 'crypto';
+
+export default function createHash(content: any) {
+  return createHashRaw('sha256').update(anyToString(content)).digest('hex');
+}
```

Comment:
I think we can get rid of this `any` with something like
```ts
export default function createHash<T extends {}>(content: T): Hash {
  return createHashRaw('sha256').update(anyToString(content)).digest('hex');
}
```

Reply:
Another small nit, we might want to rename this function as every time you try to import it it'll conflict with the `createHash` from the `crypto` package